R/aes-evaluation.r

```diff
@@ -121,7 +121,7 @@ is_calculated <- function(x) {
   } else if (is.pairlist(x)) {
     FALSE
   } else {
-    abort(glue("Unknown input: {class(x)[1]}"))
+    cli::cli_abort("Unknown input: {.cls {class(x)[1]}}")
```

Comment:
Did you add new tests if the errors weren't covered by existing tests?

Reply:
They don't have to be, but it does determine how closely I'll read the messages.